supervisor/supervisor.py

```diff
@@ -241,7 +241,7 @@ async def check_connectivity(self):
         timeout = aiohttp.ClientTimeout(total=10)
         try:
             await self.sys_websession.head(
-                "http://version.home-assistant.io/online.txt", timeout=timeout
+                "https://version.home-assistant.io/online.txt", timeout=timeout
```

Comment:
Could this cause errors when we check connectivity and the time is out of sync ?

Reply:
Correct,  but we use SSL everywhere, the whole system is not functional if the time is out of sync too much. This is our main problem for 3 years and makes that system comes never function or hang in an endless loop because we can't install or setup containers etc. This PR going to fix that close as possible for now.